test/integration/targets/setup_wildfly_server/tasks/main.yml

```diff
@@ -0,0 +1,75 @@
+- name: Install java
```

Comment:
Add the distro version check at the start of this file and use `meta: end_play` to stop the tests if the requirements aren't met. Something like:

```yaml
- name: Skip unsupported platforms
  meta: end_play
  when: CONDITION HERE
```

Reply:
added in my check list :) thanks